e2e/tests/workspace/workspace-share-modal.spec.ts

```diff
@@ -13,7 +13,7 @@ describe('Workspace Share Modal', () => {
   ];
 
   // Create new workspace with default CDR version
-  const workspace = makeWorkspaceName();
+  const workspace = 'aoutest-407581621629013'; //makeWorkspaceName();
```

Comment:
that is this?

Reply:
Good catch. I forgot to undo the change. It's used for local testing.